pkg/schema/plugins.schema.json

```
@@ -11,41 +11,46 @@
       "description": "Version of the plugins schema to which this document adheres",
       "type": "string",
       "default": "1.0.0"
-    }
-  },
-  "required": [
-    "schemaVersion"
-  ],
-  "title": "Plugins json schema",
-  "type": "object",
-  "additionalProperties": {
-    "type": "object",
-    "properties": {
-      "key": {
-        "type": "string"
-      },
-      "value": {
+    },
+    "plugins": {
+      "description": "The definition of plugins to install",
```

It would help to give people a hint that the field names under plugins should be the name of the plugin

```suggestion
      "description": "A map of plugins to install, keyed by the plugin name",
```